apps/examples/skimmer.cpp

```diff
@@ -50,7 +50,7 @@ int main(int argc, char **argv) {
 
     eventWriter->AddCurrentEvent("Events");
   }
-  cutFlowManager->SaveCutFlow();
+  cutFlowManager->SaveAllCutFlows();
```

Comment:
Can we keep the old name of this method? This is something that will require changes in all apps using the cut flow manager.